config/zones/IN-NO.yaml

```
@@ -20,7 +20,23 @@ contributors:
   - pierresegonne
   - sahitya-pavurala
   - willbeaufoy
+  - mathilde-daugy
 timezone: Asia/Kolkata
+emissionFactors:
+  direct:
+    unknown:
+      source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
+      value: 45
+  lifecycle:
+    unknown:
+      source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
+      value: 192
+isLowCarbon:
+  unknown:
+    value: 1
```

```suggestion
    value: 0.92
```
Shouldn't it be the sum of known renewable of the unknown?

---

i guess but the remainder is also other renewables according to the data source so I put 1 
I think we can put 0.92 to be on the safe side

---

Ah in that case I leave it up to you 👍🏼

---

i'll put 0.92 as we have no idea what the 8% are and are using the default unknown EF